doc/statuscodes.md

```diff
@@ -47,3 +48,5 @@ The following status codes are never generated by the library:
 - ABORTED
 - OUT_OF_RANGE
 - DATA_LOSS
+
+The decision to retry RPCs at the application level depends on the application and the type of error. There is no single guidance that will work for all. 
```

Comment:
Suggest alternative wording:

Applications that may wish to [retry](https://github.com/grpc/proposal/blob/master/A6-client-retries.md) failed RPCs must decide which status codes on which to retry.  However, because any status code returned by the gRPC library can also be returned by the server application, there is no fixed list of status codes on which it is appropriate to retry in all applications.  As a result, individual applications must make their own determination as to which status codes should cause an RPC to be retried.

Reply:
This kind of makes it seem like a status code being possibly returned by the application is the only reason why we can't give guidance on when to retry. I want to expand it to include cases such as INTERNAL status codes which might arise due to OS errors.

Reply:
Okay, so let's include something about how the gRPC library may generate the same code for different reasons, as shown in the table above.

The way this is currently phrased doesn't provide much context to the reader.

Reply:
Suggestion applied :)